### DISCUSSION TITLE
Are 3-noded shell elements supported in animation files?

### DISCUSSION THREAD

**Discussion by nawijn** [OpenRadioss/OpenRadioss#1418] [Users]
Dear all,

I have a short question: is output for 3-noded shell elements (SH3N) supported in animation files?

The reason I ask is that in the source code of the `anim_to_vtk` tool, it seems that only data for 4-noded elements is supported.

I have not tried an example yet, but maybe someone already knows the answer :).

Marco

**Comment by Nekativ** [comment]
Yes, you can output SH3N to anim files and then convert to vtk with anim_to_vtk. I can display the 3 node elements in Paraview without issue.

Maybe under the hood it treats triangle elements as degenerated quad elements?
